README.md

```diff
@@ -130,8 +130,8 @@ The following endpoints exist:
 * prod, CDN: https://epr.elastic.co
 * staging, no CDN: https://epr-staging.ea-web.elastic.dev
 * staging, CDN: https://epr-staging.elastic.co
-* experimental, no CDN: https://epr-experimental.ea-web.elastic.dev/
-* experimental, CDN: https://epr-experimental.elastic.co/
```

Comment:
Honestly I would leave it here (as it still exists/live), but would add a description on why is it deprecated.

Reply:
In that case, there is also a `7.9` endpoint IIRC. Should we document that here as well?

Reply:
yes

Reply:
Added in 2e0681981a785386de076b46b64fd57089b2d2e5.